src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Main.cs

```diff
@@ -88,7 +88,7 @@ public void Init(PluginInitContext context)
         public List<Result> Query(Query query)
         {
             List<Result> results = new List<Result>();
-            CultureInfo culture = _localizeSystemCommands ? CultureInfo.CurrentUICulture : new CultureInfo("en-US");
+            CultureInfo culture = _localizeSystemCommands ? CultureInfo.CurrentUICulture : new CultureInfo("en-US", false);
```

Comment:
@jaimecbernardo 
This isn't a test file. It's code from system plugin.

Reply:
Thanks for the notice. While it wouldn't hurt, I still reverted it ;)